components/template/package.json

```diff
@@ -29,11 +29,11 @@
   },
   "homepage": "https://github.com/supermedium/superframe/tree/components/template#readme",
   "devDependencies": {
-    "aframe": "^0.3.0",
+    "aframe": "^1.4.0",
     "browserify": "^12.0.1",
     "budo": "^7.1.0",
-    "chai": "^3.4.1",
     "chai-shallow-deep-equal": "^1.3.0",
+    "chai": "^4.3.7",
```

Comment:
karma-sinon-chai resolved to 1.3.4, which requires >= 3.5.0 < 5
Moved to latest version < 5.